### DISCUSSION TITLE
Unable to update spending limit. Unable to increase beyond 1000$.

### DISCUSSION THREAD

**Discussion by adityachowdhry-probo** [community/community#134824] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Unable to update spending limit. Unable to increase beyond 1000$. 

It asks to contact support, but we need resolution ASAP to unblock our production deployments

Any help would be appreciated. Already raised a ticket as well - [ticket](https://support.github.com/ticket/personal/0/2923410)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @adityachowdhry-probo, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It looks like a member of our support team has responded and is awaiting your reply.  We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
